lib/l10nUtil.js

```diff
@@ -248,3 +251,11 @@ const defaultReplacements = [
   [/bookmarks bar\n/g, 'bookmarks\n'],
   [/Copyright <ph name="(YEAR|year)">/g, 'Copyright © <ph name="$1">'],
 ]
+
+// Fix up some strings after aggressive first round replacement.
+const fixupReplacements = [
+  [/Brave Drive/g, 'Google Drive'],
+  [/Brave Docs/g, 'Google Docs'],
+  [/Brave Cloud Print/g, 'Google Cloud Print'],
+  [/Brave OS/g, 'Chrome OS'],
+]
```

Comment:
Should `[/Brave Safe Browsing/g, 'Google Safe Browsing'],` from the `defaultReplacements` be moved here as well?

Reply:
Moved. it would be good to have them in one place.